### ISSUE TITLE
aria-description: Newish attribute is not supported

### ISSUE THREAD

**Issue by tmcconechy** [dequelabs/axe-core#3121]
Product: axe-core

Expectation: 
Using axe jest plugin and and axe scanner in chrome we found out that using `aria-description` is reported as invalid. But it is in fact a newish valid property.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Annotations#aria_annotations_features
https://github.com/facebook/react/issues/21035

Actual:
"Invalid ARIA attribute name: aria-description" is reported

Motivation: 
Would like to use this new attribute.

<pre><code>
axe-core version: 4.12.2
axe-webdriver, extension or other integration version:  4.12.2
</code></pre>

**Comment by straker** [comment]
Thanks for the issue. [Last we checked](https://github.com/dequelabs/axe-core/issues/2078) `aria-description` wasn't well supported in most screen readers and browser combinations. However if you wan't to support it you can use `axe.configure` to allow support of them.

**Comment by tmcconechy** [comment]
OK you might have a point but was copying what is used here http://whatsock.com/Templates/Comboboxes/Native%20Inputs,%20Editable%20and%20Readonly/index.htm and it seems to work well.

How would i use axe.configure in the chrome extension?

**Comment by straker** [comment]
In the chrome extension you'll have to be logged into your deqeue account and then you'll get the functionality to configure axe. Otherwise you'd have to use axe-core directly (the JS code) to be able to do it.

**Comment by WilcoFiers** [comment]
You should use the `title` attribute.

**Comment by tmcconechy** [comment]
The title attribute is used for the tooltip. Im sort of trying to use multiple labels/attached information

**Comment by neaumusic** [comment]
My current thinking is that developers **should** add `aria-description` even if screen readers and their versions aren't caught up to the specification. TLDR I think throwing an error prevents good context from reaching users who would appreciate it -- the developer chooses between deploying code with or without errors, potentially side-lining end users.

We use the axe-core Chrome Extension and Apple's VoiceOver, which does read the aria-description. Wondering if there could be some sort of support matrix or configuration within the extension similar to [caniuse](https://caniuse.com/?search=aria-description)

**Comment by tmcconechy** [comment]
I agree @neaumusic  - it does work and it does solve our use case. So why not validate against it. Its not incorrect its just newer

**Comment by straker** [comment]
The main problem is that since not all browsers / assistive technologies support `aria-description`, there will be users that do not get the information. Since we can't say how important that information may be, we can't just allow it to pass. For example, with the following code:

```html
<input type="password" aria-label="password" aria-description="Password requirements: at least 8 characters">Hello</button>
```

* VoiceOver / Safari - does not announce the `aria-description`
* JAWS / IE11 - does not announce the `aria-description`
* JAWS / Edge - Announces the `aria-description`
* NVDA / FireFox - Announces the `aria-description`
* Talkback / Chrome (Android) - Announces the `aria-description`

This means that using IE11 and Safari, you'd miss the critical information provided by the description. Best we can do is mark it as Needs Review. Right now it looks like it's marked as a violation, so let me see if we have enough support to turn it on with Needs Review.

**Comment by WilcoFiers** [comment]
The ARIA working group hasn't even put `aria-description` out in a public working draft yet. It is experimental at best.

**Comment by woody-li** [comment]
Safari TP 153 supports `aria-description`.
https://developer.apple.com/safari/technology-preview/release-notes/
https://github.com/WebKit/WebKit/commit/7c414e7250fbc66f4800a42bb2f79b845ad1e8f0

**Comment by WilcoFiers** [comment]
I wrote a proposal on how to address both this, and the aria-braille* props here:
https://github.com/dequelabs/axe-core/issues/3899#issuecomment-1406558975

The gist is, yeah we should add them, but we'll need a rule that ensures these aren't used on element without a role or accessible name.

**Comment by padmavemulapati** [comment]
Validated with the latest axe-core-develop branch code base,
`aria-valid-attr` is now allowing the attributes `aria-braillelabel` and `aria-brailleroledescription` with these two attributes its passing.




![Image](https://user-images.githubusercontent.com/47409579/228132477-97695e91-827d-49d0-aa64-c97394480534.png)



where as in the earlier versions if we keep - `aria-braillelabel`, `aria-brailleroledescription` and `aria-description` attributess
<img width="1281" alt="image" src="https://user-images.githubusercontent.com/47409579/228132196-984803d0-281a-4ecd-9ce4-2039f5805a06.png">
